br/use-br-command-line-tool.md

```diff
@@ -501,14 +501,12 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
 
 > **Warning:**
 >
-> Although you can back up and restore system tables (such as `mysql.tidb`) using the BR tool, some unexpected situations might occur after the restore, including:
+> Although you can back up the system tables (such as `mysql.tidb`) using the BR tool, but BR does not support the restoration of the following system tables. Even if you use the `-f` setting to restore this table, they will still be ignored:
```

Comment:
Suggested change: Although you can back up system tables (such as `mysql.tidb`) using the BR tool, BR ignores the following system tables even if you use the `-f` setting to perform the restoration:

Reply:
yes